src/scenarios/billing/ListUserPaymentHistory/ListUserPaymentHistory.tsx

```diff
@@ -0,0 +1,100 @@
+import {
+  useScenarioHost,
+  VariableSearch,
+} from '@axinom/mosaic-fe-samples-host';
+import { useState } from 'react';
+import {
+  Container,
+  Divider,
+  Form,
+  Grid,
+  Header,
+  Label,
+  Segment,
+} from 'semantic-ui-react';
+import { getApolloClient } from '../../../apollo-client';
+import { getSubscriptionTransactionsQuery } from './graphql-documents';
+
+export const ListUserPaymentHistory: React.FC = () => {
+  const { activeProfile, logger } = useScenarioHost();
+  const [userAccessToken, setUserAccessToken] = useState<string>();
+
+  const apolloClient = getApolloClient(
+    new URL('graphql', activeProfile.billingServiceBaseURL).href,
+  );
+
+  const fetchTransactionHistory = async (): Promise<void> => {
+    try {
+      const result = await apolloClient.query({
+        query: getSubscriptionTransactionsQuery,
+        context: {
+          headers: {
+            Authorization: `Bearer ${userAccessToken}`,
+          },
+        },
+        fetchPolicy: 'no-cache',
+      });
+
+      logger.log(
+        'method [fetchPaymentPlansAndPaypalSettings]',
+        'output:',
+        result.data,
+      );
+    } catch (error) {
+      if (error instanceof Error) {
+        logger.error(
+          'method [fetchPaymentPlansAndPaypalSettings]',
+          'output:',
+          error.message,
+        );
+      } else {
+        logger.error(
+          'method [fetchApplicationToken]',
+          'output:',
+          JSON.stringify(error),
+        );
+      }
+    }
+  };
+
+  return (
+    <Segment basic>
+      <Header size="huge">List User Payment History</Header>
+      <Header size="small">
+        Required Services:
+        <Label>user-service</Label>
+        <Label>billing-service</Label>
```

Comment:
Change this to `billing-service & monetization-service`. Let's update other related scenarios as well because both billing and monetization services appear as a single entry in the env details station of env admin portal.